Python/ceval.c

```diff
@@ -1255,7 +1255,8 @@ positional_only_passed_as_keyword(PyThreadState *tstate, PyCodeObject *co,
 {
     int posonly_conflicts = 0;
     PyObject* posonly_names = PyList_New(0);
-
+    if (posonly_names == NULL)
+        return 1;
```

Comment:
```suggestion
        goto fail; 
```

Reply:
I don't think `goto fail` is necessary here. It only exists to decref `posonly_names`, which is NULL in this case. It will still work since it uses `Py_XDECREF`, but why go through that extra check?

Reply:
I believe that this is necessary only in order not to break the consistency of the function